source/Tutorials/Advanced/Simulators/Webots.rst

```diff
@@ -35,32 +35,16 @@ Prerequisites
 
 It is recommended to understand basic ROS principles covered in the beginner :doc:`../../../Tutorials`.
 In particular, :doc:`../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim`, :doc:`../../Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics`, :doc:`../../Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace`, :doc:`../../Beginner-Client-Libraries/Creating-Your-First-ROS2-Package` and :doc:`../../Intermediate/Launch/Creating-Launch-Files` are useful prerequisites.
-Finally, you will need to install ``webots_ros2_driver`` from a terminal with this command:
+Finally, you will need to install ``webots_ros2_driver`` from a terminal with the following commands. 
+On Windows, a WSL environment should be configured and used to execute commands. 
+This `page <https://github.com/cyberbotics/webots_ros2/wiki/Build-and-Install#windows>`_ explains how to setup such installation. 
 
-.. tabs::
-
-   .. group-tab:: Linux
-
-      .. code-block:: console
+.. code-block:: console
 
+        # On Windows, execute these in the WSL environment
```

Comment:
I find this sentence (repeated several times) is a bit confusing as it seems we should execute these lines this ONLY on Windows, not on Linux. I would remove these sentences and add a sentence earlier that explain that all the Ubuntu and ROS commands should be entered in WSL for Windows users.